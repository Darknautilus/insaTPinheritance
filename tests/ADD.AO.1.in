C circle 1 2 3
R rectangle 1 2 3 4
L line 1 3
OA agregated circle rectangle line
