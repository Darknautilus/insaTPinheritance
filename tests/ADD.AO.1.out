OK
OK
OK
OK
OA agregated circle line rectangle
C circle 1 2 3
L line 1 3 5 6
R rectangle 1 2 3 4
